packages/account/src/BiconomySmartAccount.ts

```diff
@@ -102,15 +88,17 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
 
   async getSmartAccountAddress(accountIndex: number = 0): Promise<string> {
     try {
+      // TODO: may be create a state to manage address
```

Comment:
Agreed. we should keep this state where we can get the address from user EOA and index.

Reply:
resolved